geoips/geoips_utils.py

```diff
@@ -75,40 +75,30 @@ def load_all_yaml_plugins():
     in ``.yaml``. Read each plugin file
     """
     # Load all entry points for plugin packages
-    plugin_packages = get_entry_point_group("geoips.plugin_packages")
 
-    # Loop over the plugin packages and load all of their yaml plugins
+    reg_plug_path = resources.files("geoips") / "registered_plugins.yaml"
```

Comment:
@evrose54 @jsolbrig Note I moved this into the package directory (it was ../registered_plugins.yaml).  I believe this will be necessary when we pip install in site-packages, correct?  Let me know if you think this will break anything.

Reply:
This looks correct to me!